flink-runtime/src/main/java/org/apache/flink/runtime/blob/BlobUtils.java

```diff
@@ -150,8 +150,7 @@ static File initLocalStorageDirectory(String basePath) throws IOException {
 		File storageDir;
 
 		// NOTE: although we will be using UUIDs, there may be collisions
-		final int MAX_ATTEMPTS = 10;
-		for(int attempt = 0; attempt < MAX_ATTEMPTS; attempt++) {
+		for (int attempt = 0; attempt < 10; attempt++) {
```

Comment:
Should we keep and simply rename the constant?

Reply:
ok, sure - I admit, I was tempted to change it and went for it - let me change that back